### PR TITLE
Fix the revocable proxy topic

### PR DESCRIPTION
**Pull request by Therrance**
https://github.com/enkidevs/curriculum/commit/7477f8a301aa317ecf8c182c3f30c43e6a33876b

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/dummy-workout/revokable-proxies-and-usages.md?ref=d25c1c0ef52345b7f2134c25838efb1d06fa4b05
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md?ref=d25c1c0ef52345b7f2134c25838efb1d06fa4b05

File: javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Proxies allow you to intercept many different operations and methods:
 * ownKeys
 * apply
 
-Of course you only need to specify those you want to intercept otherwise the default behavior will occur.
+Of course you only need to specify those you want to intercept otherwise the default behaviour will occur.
```
Comment (nem035) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: Hi @Therrance, this isn't really a needed change because it comes down to the [difference between British vs American English](https://www.grammar.com/behavior_vs._behaviour) and since a majority of programming languages and literature follow the American English conventions, that's what we're doing as well.

File: javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ Some potential usages:
 Fill in the gaps such that log statements hold true:
 
 ```javascript
-const revocable = ???.???({}, {
+const rev = ???.???({}, {
```
Comment (nem035) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: Just wondering, why this change?

Comment (nem035) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: Why this change when the rest of the example is using the `revocable` variable?

Comment (nem035) [reply javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: You should also change this line too:

```js
// change this
const proxy = revocable.???;

// to
const proxy = rev.???
```

Comment (nem035) [reply javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: For this snippet:

```js
const rev = ???.???({}, {
  ???: () => 42
});

const proxy = rev.???;
console.log(proxy.enki)
// 42

???.???()
console.log(proxy.enki)
// TypeError
```

The top correct answers should be:

```md
* Proxy
* revocable
* get
* proxy
* rev
* revoke
```

**Comment by ryanwarsaw** [comment]
Closing due to stale changes, the remote has been deleted so I can't manually go in and make the appropriate changes to merge this.
